Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -209,11 +209,6 @@ - (void)URLSession:(__attribute__((unused)) NSURLSession *)session
        summary:[NSString stringWithFormat:@"Error parsing problem doc downloading %@ book", book.distributor]
        metadata:@{ @"book": [book loggableShortString] }];
     }
-    [self logBookDownloadFailure:book
```

Comment:
this was a dupe. The actual logging is at the bottom of the (very long) function.